stips/astro_image/astro_image.py

```diff
@@ -668,68 +580,49 @@ def make_epsf_array(self, psf_type = 'normal'):
-            # Second limit -- PSF no larger than half of max convolution area.
-            conv_size = self.convolve_size // (2*self.oversample)
-            # Third limit -- prevent aliasing
-            limit = 60.
-            if psf_type == 'bright': limit = 120.
-            elif psf_type == 'xbright': limit = 240.
-            safe_size = int(np.floor(limit * self.photplam / (2 * self.scale[0])))
-            if safe_size <= 0:
-                safe_size = max(self.xsize, self.ysize)
-            msg = "PSF choosing between {}, {} and {}"
-            self._log("info", msg.format(ins_size, conv_size, safe_size))
-            fov_pix = min(ins_size, conv_size, safe_size)
+            fov_pix = PSF_BOXSIZE
+            if psf_type == 'bright': fov_pix = PSF_BRIGHT_BOXSIZE
+            elif psf_type == 'xbright': fov_pix = PSF_EXTRA_BRIGHT_BOXSIZE
             if fov_pix%2 == 0:
                 fov_pix += 1
             num_psfs = self.psf_grid_size*self.psf_grid_size
             #num_psfs = 9#self.psf_grid_size*self.psf_grid_size
             if SelectParameter('psf_cache_enable'):
                 save = True
                 overwrite = True
-                psf_file = "{}psf_{}_{}_{}_{}_{}".format(prefix, 
-                                                         self.instrument,
-                                                         stips_version, 
-                                                         self.filter,
-                                                         self.oversample,
-                                                         self.psf_grid_size)
+                psf_file = "{}psf_{}_{}_{}".format(prefix, 
+                                                   self.instrument,
+                                                   stips_version, 
+                                                   self.filter)
             else:
                 save = False
                 overwrite = False
                 psf_dir = None
                 psf_file = None
-            msg = "{}: Starting {}x{} PSF Grid creation at {}"
-            self._log("info", msg.format(self.name, self.psf_grid_size, 
-                                         self.psf_grid_size, time.ctime()))
+            msg = "{0}: Starting {1}x{1} PSF Grid creation at {2}"
+            self._log("info", msg.format(self.name, self.psf_grid_size, time.ctime()))
             # Supersample the pixel scale to get WebbPSF to output
             # PSF models with even supersamplingcentered at the center of a pixel
-            ins.pixelscale = scale / 4
+            ins.pixelscale = scale / PSF_UPSCALE
             self.psf = ins.psf_grid(all_detectors=False, num_psfs=num_psfs,
                                     fov_pixels=fov_pix, normalize='last',
-                                    oversample=self.oversample, save=save,
+                                    oversample=PSF_UPSCALE, save=save,
```

Comment:
oversample ≠ PSF_UPSCALE, this should be set to 1

Reply:
I fixed this without accepting this change.